### PR TITLE
make send asset a modal

### PR DESCRIPTION
**Pull request by matjazc**
New feature:
Make send asset a modal

- Add a send asset button on the account added
- When clicking on the Send button, the send asset modal will be open
- Modal can be closed now
- Remove Send assets from side nav

### CODE REVIEW

File: packages/app-accounts/src/Account.tsx [hunk 1]
```diff
@@ -316,6 +320,17 @@ function Account ({ address, className, filter, isFavorite, toggleFavorite }: Pr
...
       </td>
+      <td className='number middle'>
+        <Button
+          icon='paper plane'
+          isPrimary
+          key='send'
+          label={t('Send')}
+          onClick={() => toggleTransfer()}
+          size='small'
+          tooltip={t('Send funds to this address')}
```
Comment (holygits) [inline packages/app-accounts/src/Account.tsx:331]: "Send funds from this account"

File: packages/app-accounts/src/modals/TransferWithType.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+  return (
+    <Modal className='app--accounts-Modal' header={t('Send funds')}>
+      <Modal.Content>
+        <div className={className}>
+          <InputAddress
+            defaultValue={propSenderId}
+            help={t('The account you will send funds from.')}
+            isDisabled={!!propSenderId}
+            label={t('send from account')}
```
Comment (holygits) [inline packages/app-accounts/src/modals/TransferWithType.tsx:90]: To be consistent we should capitalise this
"**S**end from account"

File: packages/app-accounts/src/modals/TransferWithType.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+            onChange={setSenderId}
+            type='account'
+          />
+          <InputAddress
+            defaultValue={propRecipientId}
+            help={t(
+              'Select a contact or paste the address you want to send funds to.'
+            )}
+            isDisabled={!!propRecipientId}
+            label={t('send to address')}
```
Comment (holygits) [inline packages/app-accounts/src/modals/TransferWithType.tsx:102]: Same here: "**S**end to address"

**Review by holygits — COMMENTED** [review]
few small things but otherwise lgtm.
will need to update the landing page to remove the "send assets" link

**Comment by KarishmaBothara** [comment]
need to fix Lint & build errors.. code lgtm

**Comment by matjazc** [comment]
> need to fix Lint & build errors.. code lgtm

Karishma, this was immediately fixed, there is no error anymore

**Review by KarishmaBothara — APPROVED** [review]

**Pull request merged**